python/lsst/afw/table/schema.cc

```diff
@@ -177,6 +177,11 @@ PYBIND11_PLUGIN(_schema) {
                                            std::string const &,
                                            std::string const &) const) &Schema::join,
                   "a"_a, "b"_a, "c"_a, "d"_a);
+    clsSchema.def("__repr__", [](Schema const & self) {
+        std::ostringstream os;
+        os << self;
+        return os.str();
+    });
```

Comment:
We will want to have some policy on `__str__` vs `__repr__` in particular I think the latter should always include the type because pybind11 prints `__repr__` when overload resolution fails to find a candidate.